### PR TITLE
feature/marquee-section

### PR DESCRIPTION
**Pull request by phongduong-dp**
# Description

Create marquee section for logos

# Checklist

- [x] This is a breaking changes?

### CODE REVIEW

File: packages/core-components/src/components/sections/MarqueeSection.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import Marquee from 'react-fast-marquee';
+
+export type MarqueeContentType = {
+  id: string;
+  url: string;
+  alt: string;
+};
+
+type MarqueeSectionProps = {
+  content?: MarqueeContentType[];
```
Comment (its-rav) [inline packages/core-components/src/components/sections/MarqueeSection.tsx:11]: - sửa lại naming cho list
- sao cái này nullable ta

File: packages/core-components/src/components/sections/MarqueeSection.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      {content &&
+        content.map((item) => (
+          <div
+            className='flex items-center justify-center text-center mx-6 md:w-44 md:h-16 sm:w-36 sm:h-12'
+            key={item.id}
+          >
+            <img src={item.url} alt={item.alt} className='w-full h-full' />
+          </div>
+        ))}
+      {children}
```
Comment (its-rav) [inline packages/core-components/src/components/sections/MarqueeSection.tsx:52]: children có cần thiết k bạn

**Review by its-rav — CHANGES_REQUESTED** [review]
can I have a video showing how this component should've render?

**Comment by phongduong-dp** [comment]
> can I have a video showing how this component should've render?

Hi @its-rav  please see below videos.

**Default**

https://github.com/its-rav/namvietstore-fe/assets/92157056/f219d5f0-c1b4-4b98-9c31-3cb761153988

**No Logo**

https://github.com/its-rav/namvietstore-fe/assets/92157056/ce4c552c-e69e-41ee-b381-a97983a6a65e

**Small screen size**

https://github.com/its-rav/namvietstore-fe/assets/92157056/b36af666-1aa8-495c-8425-91020f5657bd

File: packages/core-components/src/components/sections/index.ts [hunk 3]
```diff
@@ -1,10 +1,15 @@
+import MarqueeSection, {
+  type MarqueeItemsType,
+} from '@/components/sections/MarqueeSection';
 import StatisticSection, {
   type StatisticItemsType,
   type StatisticSectionProps,
 } from '@/components/sections/StatisticSection';
 
 export {
+  type MarqueeItemsType,
```
Comment (its-rav) [inline packages/core-components/src/components/sections/index.ts:10]: ...Props fren

File: packages/core-components/src/components/sections/index.ts [hunk 4]
```diff
@@ -1,10 +1,15 @@
+import MarqueeSection, {
+  type MarqueeItemsType,
+} from '@/components/sections/MarqueeSection';
 import StatisticSection, {
   type StatisticItemsType,
```
Comment (its-rav) [inline packages/core-components/src/components/sections/index.ts:5]: đây nữa nè, ủa bữa chưa fix hả, mà nó phải là StatisticItemType chứ ta

**Review by its-rav — CHANGES_REQUESTED** [review]

**Review by its-rav — APPROVED** [review]

**Pull request merged**